src/tbbmalloc/backend.h

```diff
@@ -77,7 +77,7 @@ class MemExtendingSema {
                     break;
                 }
             } else {
-                SpinWaitWhileEq(active, prevCnt);
+                tbb::detail::spin_wait_while_eq(active, prevCnt);
```

Comment:
It does not seems as a good approach because it brings uncontrolled dependency on TBB